Assets/MRTK/Providers/UnityAR/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "com.microsoft.mixedreality.toolkit.unityar",
+    "displayName": "Mixed Reality Toolkit Mobile AR Provider",
+    "version": "2.5.0-preview.0",
+    "unity": "2018.4",
+    "description": "Platform support for mobile device AR on the Microsoft Mixed Reality Toolkit",
+    "keywords": [
+        "mrtk",
+        "microsoft",
+        "mixed",
+        "reality",
+        "augmented",
+        "virtual",
+        "xr",
+        "mobile",
+        "android",
+        "ios"
+    ],
+    "author": "Microsoft Corporation",
+    "license": "MIT",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/microsoft/MixedRealityToolkit-Unity.git"
+    },
+    "bugs": {
+        "url": "https://github.com/microsoft/MixedRealityToolkit-Unity/issues"
+      },
+    "dependencies": {

```

Comment:
Should we take a dependency on whichever arsubsystems / arfoundation min version we require?

Reply:
we probably should. thx

Reply:
done. added the known good version from 2018.4. 2019.4 should update appropriately based on past experience